src/components/alertCo.js

```diff
@@ -0,0 +1,68 @@
+import React, { useState } from "react"
+import PropTypes from "prop-types"
+
+import { makeStyles } from "@material-ui/core/styles"
+import Alert from "@material-ui/lab/Alert"
+import AlertTitle from "@material-ui/lab/AlertTitle"
+import Collapse from "@material-ui/core/Collapse"
+import CloseIcon from "@material-ui/icons/Close"
+import IconButton from "@material-ui/core/IconButton"
+
+const useStyles = makeStyles({
+  root: {
+    backgroundColor: ({ customColor }) => customColor,
+    fontSize: ({ fontSize }) => fontSize || "1.25rem",
+    fontWeight: ({ textBold }) => (textBold ? 500 : 300),
+    width: ({ width }) => width || "100%",
+    padding: [11, 24, 13, 16],
+  },
+  action: {
+    alignItems: "baseline",
+  },
+})
+
+const AlertCo = (props) => {
+  const { children, alertTitle, closeIcon } = props
```

Comment:
Could you do this in this way? to prevent pass the unused props to the theme
```suggestion
const AlertCo = ({ alertTitle, children, closeIcon, ...props }) => {
```

Reply:
done